windows/src/global/delphi/general/CompilePackage.pas

```diff
@@ -170,14 +170,26 @@ function TCompilePackage.BuildKMP: Boolean;
 
     kmpinf.Assign(pack);
     kmpinf.RemoveFilePaths;
-    kmpinf.FileName := FTempPath + '\kmp.inf';
 
     psf := TPackageContentFile.Create(kmpinf);
     psf.FileName := 'kmp.inf';
     psf.Description := 'Package information';
     psf.CopyLocation := pfclPackage;
+
+    kmpinf.Files.Add(psf);
+
+    psf := TPackageContentFile.Create(kmpinf);
```

Comment:
Is `kmpinf` named for "kmp.inf" or "kmp information"?
Since you're using it to generate both .inf and .json, maybe rename it to a general `kmpinfo`?

Reply:
Because `kmpinf` is of type `TKMPInfFile`, I'll leave it. It can be "kmp information"...